docs/concepts/automations.md

```diff
@@ -138,14 +134,12 @@ Prefect tries to infer the relevant event whenever possible, but sometimes one d
 
 Specify a name and, optionally, a description for the automation.
 
-![Configuring details for an automation in Prefect Cloud.](/img/ui/automations-details.png)
-
-
 ## Create an automation via deployment triggers
 
 To enable the simple configuation of event-driven deployments, Prefect provides deployment triggers - a shorthand for creating automations that are linked to specific deployments to run them based on the presence or absence of events.
 
-To 
+To
```

Comment:
I know you didn't introduce this issue, but this looks like a fragment that should be removed.

Reply:
good catch